<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobInfo</class>
 <widget class="QWidget" name="JobInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_10">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,20">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="10,1,0">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="apply">
           <property name="text">
            <string>应用</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_2">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,20">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>基本信息</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <property name="styleSheet">
                <string notr="true">border: 1px solid grey;
border-radius: 15px; </string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="styleSheet">
                       <string notr="true">border: none</string>
                      </property>
                      <property name="text">
                       <string>任务名:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="jobName">
                      <property name="styleSheet">
                       <string notr="true">border: none</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="styleSheet">
                       <string notr="true">border: none</string>
                      </property>
                      <property name="text">
                       <string>需求内存:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="ram">
                      <property name="styleSheet">
                       <string notr="true">border: none</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_3">
                      <property name="styleSheet">
                       <string notr="true">border: none</string>
                      </property>
                      <property name="text">
                       <string>GB</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="styleSheet">
                       <string notr="true">border: none</string>
                      </property>
                      <property name="text">
                       <string>周期:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="period"/>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="styleSheet">
                       <string notr="true">border: none</string>
                      </property>
                      <property name="text">
                       <string>s</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_14">
                    <item>
                     <widget class="QLabel" name="label_15">
                      <property name="styleSheet">
                       <string notr="true">border: none</string>
                      </property>
                      <property name="text">
                       <string>截止:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="lineEdit"/>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_16">
                      <property name="styleSheet">
                       <string notr="true">border: none</string>
                      </property>
                      <property name="text">
                       <string>s</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_13">
                    <item>
                     <widget class="QLabel" name="label_14">
                      <property name="styleSheet">
                       <string notr="true">border: none</string>
                      </property>
                      <property name="text">
                       <string>主机:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="host"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>计算需求</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="gputable"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="10,1">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="styleSheet">
                <string notr="true">border: none
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../img/加.png</normaloff>../img/加.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,20">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>任务运行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="showjob" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
